parser_connector.py

```diff
@@ -15,21 +15,21 @@
 #
 #
 # Phantom App imports
-import phantom.app as phantom
-from phantom.base_connector import BaseConnector
-from phantom.action_result import ActionResult
-import phantom.rules as ph_rules
-
-import sys
-import json
+import calendar
 import email
+import json
+import sys
 import threading
-import parser_email
-import parser_methods
 import time
-import calendar
+
```

Comment:
The changes in this file are:
* done as a result of running `pre-commit run --all-files`
* Manually update the too-long lines.